requirements/base.txt

```diff
@@ -1,9 +1,9 @@
 # SHA1:082d6bf55d8a06cdc1dd3a0a78dd31c318c26424
 #
-# This file is autogenerated by pip-compile-multi
+# This file was generated by pip-compile-multi.
 # To update, run:
 #
-#    pip-compile-multi
+#    requirements upgrade
```

Comment:
Not important but I wonder why this changed? Were there updates in pip-compile-multi?